yarn.lock

```diff
@@ -0,0 +1,4775 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Do we need `package-lock.json` and `yarn.lock`? Can we just stick with one instead?